massa-client/src/cmds.rs

```diff
@@ -613,6 +627,59 @@ impl Command {
                 }
                 Ok(Box::new(()))
             }
+            Command::send_smart_contract => {
+                if parameters.len() != 6 {
+                    bail!("wrong param numbers");
```

Comment:
wrong number of parameters

Reply:
I count 6 params :confused:

Reply:
I was just improving the error message :)

Reply:
ah ok x)